tests/cases/compiler/throwType_property_access.ts(4,1): error TS2809: Type instantiated results in a throw type saying: 
    


==== tests/cases/compiler/throwType_property_access.ts (1 errors) ====
    class X<T> {
        i: T extends number ? throw '' : string = {} as any
    }
    new X<0>().i
    ~~~~~~~~~~~~
!!! error TS2809: Type instantiated results in a throw type saying: 
!!! error TS2809:     